worker/dbaccessor/worker.go

```diff
@@ -548,7 +548,7 @@ func (w *dbWorker) shutdownDqlite(ctx context.Context) {
 		return
 	}
 
-	ctx, cancel := context.WithTimeout(context.TODO(), time.Second*30)
+	ctx, cancel := context.WithTimeout(ctx, time.Second*30)
```

Comment:
@manadart we need to ensure that we pass the context from the tomb here.

Reply:
Yeah, but we have a catacomb rather than a tomb. We'll need to work something out.